website/cue/reference/components/enrichment_tables/file.cue

```diff
@@ -0,0 +1,141 @@
+package metadata
+
+components: enrichment_tables: file: {
```

Comment:
None of this configuration makes sense for enrichment tables, hence my comment above. Things like `title`, `classes`, `support`, etc just don't make sense in this context. I'd prefer to rework the docs so they are reflective of the actual feature and not wedged in as a component which feels awkward.